extensions/aave-search/CHANGELOG.md

```diff
@@ -1,8 +1,16 @@
 # Aave Search Changelog
 
-## [Initial Version] - 2025-03-21
+## [1.1.0] - {PR_MERGE_DATE}
+
+### Offline fallback & Fixes
+
+- Store previously fetched values for offline use
+- Fixed sorting error that was causing extension crashes. Kudos to @StErMi.
```

Comment:
```suggestion
- Fixed sorting error that was causing extension crashes. Kudos to [@StErMi](https://github.com/StErMi).
```

How about this? The readme cannot convert the mention to a link. But we could manually make it to a link.